ckcp/run.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+#quit if exit status of any cmd is a non-zero value
+set -exuo pipefail
+
+#create ns, sa, deployment and service resources
+#check if namespace and serviceaccount exists; if not, create them
+NS=$(kubectl get namespace ckcp --ignore-not-found);
+if [[ "$NS" ]]; then
+  echo "namespace ckcp exists";
+  kubectl delete all --all -n ckcp;
+  kubectl delete ns ckcp;
+  kubectl create namespace ckcp;
+else
+  echo "creating namespace ckcp";
+  kubectl create namespace ckcp;
+fi;
+SA=$(kubectl get sa anyuid -n ckcp --ignore-not-found);
+if [[ "$SA" ]]; then
+  echo "service account anyuid already exists in ckcp namespace";
+else
+  echo "creating service account anyuid in ckcp namespace";
+  oc create sa anyuid -n ckcp;
+  oc adm policy add-scc-to-user -n ckcp -z anyuid anyuid;
+fi;
+
+sed "s|quay.io/bnr|$KO_DOCKER_REPO|g" config/kcp-deployment.yaml | kubectl apply -f -
+kubectl apply -f config/kcp-service.yaml
+
+#check if kcp inside pod is running or not
+end=$(($SECONDS+240))
+while [ $SECONDS -lt $end ] && [ "$(kubectl get pods -n=ckcp -l=app='kcp-in-a-pod' -o jsonpath='{.items[*].status.containerStatuses[0].ready}')" != "true" ]; do
```

Comment:
There is kubectl command for that :)
`kubectl wait --for=condition=Ready pod/busybox1`

Reply:
Neat :)
Have used this instead of the while loop.